docs/Node Operators/node-versions.md

```diff
@@ -12,6 +12,77 @@ metadata:
+
+- Added the [`ETH_USE_FORWARDERS` config](/docs/configuration-variables/#eth_use_forwarders) option to enable transactions forwarding contracts.
+
+- In the `directrequest` job pipeline, three new block variables are available:
+  - `$(jobRun.blockReceiptsRoot)` : the root of the receipts trie of the block (hash)
+  - `$(jobRun.blockTransactionsRoot)` : the root of the transaction trie of the block (hash)
+  - `$(jobRun.blockStateRoot)` : the root of the final state trie of the block (hash)
+
+- `ethtx` tasks can now be configured to error if the transaction reverts on-chain. You must set `failOnRevert=true` on the task to enable this behavior:
+
+    `foo [type=ethtx failOnRevert=true ...]`
+
+    The `ethtx` task now works as follows:
+
+    - If `minConfirmations == 0`, task always succeeds and nil is passed as output.
+    - If `minConfirmations > 0`, the receipt is passed through as output.
+    - If `minConfirmations > 0` and `failOnRevert=true` then the `ethtx` task will error on revert.
+    - If `minConfirmations` is not set on the task, the chain default will be used which is usually 12 and always greater than 0.
+
+- `http` task now allows specification of request headers. Use it like the following example:
+
+    `foo [type=http headers="[\\"X-Header-1\\", \\"value1\\", \\"X-Header-2\\", \\"value2\\"]"]`.
+
+### Fixed
+
+- Fixed `max_unconfirmed_age` metric. Previously this would incorrectly report the max time since the last rebroadcast, capping the upper limit to the EthResender interval. This now reports the correct value of total time elapsed since the _first_ broadcast.
+
+- Correctly handle the case where bumped gas would exceed the RPC node's configured maximum on Fantom. Note that node operators should check their Fantom RPC node configuration and remove the fee cap if there is one.
+
+- Fixed handling of the Metis internal fee change.
+
+### Removed
+
+- The `Optimism` OVM 1.0 `GAS_ESTIMATOR_MODE` has been removed and the `Optimism2` `GAS_ESTIMATOR_MODE` has been renamed to `L2Suggested`.
+
+- `MIN_OUTGOING_CONFIRMATIONS` has been removed and no longer has any effect. The [`ETH_FINALITY_DEPTH` environment variable](/docs/configuration-variables/#eth_finality_depth) is now used as the default for `ethtx` confirmations instead. You can override this on a per-task basis by setting `minConfirmations` in the task definition. For example, `foo [type=ethtx minConfirmations=42 ...]`.
+
+    This setting might have a minor impact on performance for very high throughput chains. If you don't care about reporting task status in the UI, set `minConfirmations=0` in your job specs. For more details, see the [relevant section of the performance tuning guide](https://www.notion.so/chainlink/EVM-performance-configuration-handbook-a36b9f84dcac4569ba68772aa0c1368c#e9998c2f722540b597301a640f53cfd4).
```

Comment:
Was the indentation intentional?

Reply:
Yes this keeps the paragraph as part of the bullet point.